### PR TITLE
Ps 10805 add a switch for using the v2 api in some commands

### PR DESCRIPTION
**Pull request by BartoszCki**

### CODE REVIEW

File: gradient/api_sdk/clients/deployment_client.py [hunk 1]
```diff
@@ -59,10 +57,13 @@ def start(self, deployment_id):
             gradient deployments start --id <your-deployment-id>
 
         :param str deployment_id: Deployment ID
+        :param bool use_vpc:
         """
-        return self._get_post_response(deployment_id)
 
-    def stop(self, deployment_id):
+        repository = repositories.StartDeployment(api_key=self.api_key, logger=self.logger)
+        repository.start(deployment_id, use_vpc=use_vpc)
```
Comment (kossak) [inline gradient/api_sdk/clients/deployment_client.py]: no return here?

File: gradient/api_sdk/clients/experiment_client.py [hunk 2]
```diff
@@ -161,7 +155,7 @@ def create_multi_node(
 
         :param str name: Name of new experiment  [required]
         :param str project_id: Project ID  [required]
-        :param str experiment_type_id: Experiment Type ID [GRPC|MPI] [required]
+        :param int experiment_type_id: Experiment Type ID [GRPC|MPI] [required]
```
Comment (kossak) [inline gradient/api_sdk/clients/experiment_client.py]: GRPC/MPI ain't int

File: gradient/api_sdk/clients/experiment_client.py [hunk 3]
```diff
@@ -367,7 +367,7 @@ def run_multi_node(
...
```
Comment (kossak) [inline gradient/api_sdk/clients/experiment_client.py]: same as above, unless you parse input (which is string - GRPC or MPI), this is not int

File: gradient/api_sdk/clients/deployment_client.py (hunk 1 above)
Comment (BartoszCki) [reply gradient/api_sdk/clients/deployment_client.py]: nope. An exception is raised when command failed

File: gradient/api_sdk/clients/experiment_client.py (hunk 2 above)
Comment (BartoszCki) [reply gradient/api_sdk/clients/experiment_client.py]: Sure. But the experiment type id is. Fixing the comment

File: gradient/api_sdk/clients/experiment_client.py (hunk 3 above)
Comment (BartoszCki) [reply gradient/api_sdk/clients/experiment_client.py]: Fixing the comment

**Pull request merged**